src/main/java/teammates/logic/api/GateKeeper.java

```diff
@@ -396,36 +396,36 @@ private boolean isStudent() {
      * Verifies that the action is accessible when the user is either an instructor of the course, a student of the course
      * or his/her team member, or an admin.
      */
-    public void verifyAccessibleForCurrentUserAsInstructorOrTeamMemberOrAdmin(AccountAttributes account, String courseId,
+    public void verifyAccessibleForCurrentUserAsInstructorOrTeamMemberOrAdmin(String googleId, String courseId,
```

Comment:
I suppose @rrtheonlyone will also use the method?

Reply:
Yes, and we have discussed about changing attribute to googleId